cumulusci/utils/git.py

```diff
@@ -71,6 +72,9 @@ def parse_repo_url(url: str) -> Tuple[str, str, str]:
     Tuple: (str, str, str)
         Returns (owner, name, host)
     """
+    if not url:
+        raise error("Url is none or must have `remote` set as origin")
```

Comment:
The error message could be clearer. Consider rephrasing it to "The provided URL is empty or invalid."